src/containers/control_form/_control_form.scss

```diff
@@ -64,6 +64,11 @@
   padding: 0 15px;
   margin: 10px auto;
 
+  &[active="true"] {
```

Comment:
I assume this is for the active palette?
- Thank you for tackling this, I was thinking we wouldn't be able to get to it